deployment/ccip/operation/router.go

```diff
@@ -52,6 +55,7 @@ func deployRouter(b operations.Bundle, deps TonDeps, in DeployRouterInput) (Depl
 			Owner:        deps.TonChain.WalletAddress,
 			PendingOwner: nil,
 		},
+		WrappedNative: TonTokenAddr,
```

Comment:
Add it here https://github.com/smartcontractkit/chainlink-ton/blob/main/deployment/ccip/sequence/deploy_ccip.go#L88-L97 and remove the deployRouter operation that is not used anymore